### DISCUSSION TITLE
.env for development only, I  stumbled on an  error

### DISCUSSION THREAD

**Discussion by DCVglobalnetwork** [Cyfrin/foundry-full-course-cu#2279] [Q&A]
Dear Developers 

![image](https://github.com/user-attachments/assets/521ca5bb-e1ef-4bb8-913c-00b9b1681c45)

Please help. Is it correct that it did not deploy with my private key or should it still deploy? because if my private key was not found by throwing an error,? Thanks for getting back

**Comment by EngrPips** [comment]
Hello @DCVglobalnetwork, Please don't show your private key in public like this, even though it is a dummy private key. I think the command is suppose to be the below 

```js
forge script script/DeploySimpleStorage.s.sol:DeploySimpleStorage --rpc-url $(SEPOLIA_RPC_URL) --private-key $(SEPOLIA_PRIVATE_KEY) --broadcast
```

**Reply by DCVglobalnetwork** [reply]
I understand that even with fake private keys yes, I was hesitant but I eventually posted with. I appreciate your honesty.
Yes I now recognise the command from previous projects. Thank you

**Reply by EngrPips** [reply]
You are welcome, Friend. Is your issue resolved now?

**Reply by DCVglobalnetwork** [reply]
And with anvil? 
```sh
forge script script/DeploySimpleStorage.s.sol:DeploySimpleStorage --rpc-url $RPC_URL  --broadcast --private-key $PRIVATE_KEY 
```

**Reply by EngrPips** [reply]
I think the command is the same for all chain if that is what you are asking.

**Reply by DCVglobalnetwork** [reply]
yes , I did it, Thank you but now I am facing another small issue . I went to the documentactions and copy the repo and denied permission
could you please help me with this issue? 

![image](https://github.com/user-attachments/assets/aace2804-20d3-4f48-a50a-42b01741f734)

**Reply by EngrPips** [reply]
Hey @DCVglobalnetwork use the below code 

```js
git clone https://github.com/matter-labs/foundry-zksync
```

**Reply by DCVglobalnetwork** [reply]
thank you

**Reply by EngrPips** [reply]
You are welcome.
